Source/WebCore/page/SecurityOriginData.h

```diff
@@ -42,6 +49,14 @@ struct SecurityOriginData {
     {
         RELEASE_ASSERT_WITH_SECURITY_IMPLICATION(!isHashTableDeletedValue());
     }
+    SecurityOriginData(MarkableOpaqueOriginIdentifier opaqueOriginIdentifier)
```

Comment:
Seems like this should take an `OpaqueOriginIdentifier` and be marked as explicit?